bom/application/pom.xml

```diff
@@ -1842,6 +1842,16 @@
                 <artifactId>quarkus-jaxrs-client-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-rest-client-microprofile</artifactId>
```

Comment:
So is the idea to have both this and the RESTEasy based client around?

Reply:
I think we need some bikeshedding on the name. And also the name of the directory of the module looks wrong to me. We don't have `quarkus-` anywhere else in directory names.

Reply:
Ideally, I'd like to rename the new thing to `quarkus-rest-client` and the old to sth like `quarkus-rest-client-resteasy`. But then people who'd update would start using the new thing unknowingly...
maybe it should be something like `quarkus-resteasy-reactive-rest-client` for now?

Reply:
@gsmet I changed the directory name for rest client not to start from `quarkus` (now). But all other resteasy-reactive extensions start with `quarkus`.